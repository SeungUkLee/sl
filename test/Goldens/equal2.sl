(* equal (function) *)

let f = fun x -> x + 1 in
let g = fun x -> true in
f == g

(* result : type error *)
