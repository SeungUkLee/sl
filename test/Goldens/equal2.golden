[Error] "g" expression has type "bool" but an expression was expected of type "int"
  expected: int
    actual: bool
